docs-ui/stories/components/checkbox.stories.js

```diff
@@ -19,52 +20,36 @@ export default {
     disabled: {
       control: {type: 'boolean'},
     },
+    checked: {
+      control: {type: 'boolean'},
+    },
   },
 };
 
 export const Default = props => {
-  const [checked, setChecked] = useState(true);
+  const [{checked}, updateArgs] = useArgs();
 
   return (
     <div>
-      <Checkbox
-        {...props}
-        checked={checked}
-        onChange={e => setChecked(e.target.checked)}
-      />
+      <Checkbox {...props} onChange={e => updateArgs({checked: !!e.target.checked})} />
     </div>
   );
 };
 
 export const WithLabel = props => {
-  const [check1, setCheck1] = useState(true);
-  const [check2, setCheck2] = useState(false);
+  const [{checked}, updateArgs] = useArgs();
 
   return (
-    <div>
-      <Label>
-        Label to left
-        <Checkbox
-          {...props}
-          checked={check1}
-          onChange={e => setCheck1(e.target.checked)}
-        />
-      </Label>
```

Comment:
I don't think we want to encourage putting labels on the left side of checkboxes.